airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/StatusMainInfo.tsx

```diff
@@ -59,12 +58,7 @@ export const StatusMainInfo: React.FC<StatusMainInfoProps> = ({
       </div>
       {connection.status !== ConnectionStatus.deprecated && (
         <div className={styles.enabledControlContainer}>
-          <EnabledControl
-            onStatusUpdating={onStatusUpdating}
-            disabled={!allowSync}
-            connection={connection}
-            frequencyType={getFrequencyType(connection.scheduleData?.basicSchedule)}
```

Comment:
This can be figured out directly in the child component now.  
https://github.com/airbytehq/airbyte/issues/15609 tracks converting the child component to use the service and remove the need to send `connection` as a prop at all.